packages/console/src/pages/EnterpriseSsoDetails/Connection/SamlMetadataForm/index.tsx

```diff
@@ -58,10 +83,36 @@ function SamlMetadataFormFields({
             />
           </FormField>
           <FormField isRequired title="enterprise_sso.metadata.saml.certificate_field_name">
-            <Textarea
-              {...register('x509Certificate', { required: true })}
-              placeholder={t('enterprise_sso.metadata.saml.certificate_placeholder')}
-              error={Boolean(errors.x509Certificate)}
+            <Controller
+              control={control}
+              name="x509Certificate"
+              rules={{
+                validate: (value) => {
+                  if (!value) {
+                    return globalTranslate('errors:single_sign_on.certificate_required');
```

Comment:
Is this error i18n key shared between console and core? Do we have use cases like this before?
Maybe we can use another `useTranslation` hook with `errors` key prefix?

Reply:
Updated.